extensions/wikia/WikiaMaps/js/WikiaMapsUtils.js

```diff
@@ -280,12 +280,15 @@ define(
 		 * @param {function} cb - callback function to be called after login
 		 */
 		function showForceLoginModal(origin, cb) {
-			w.UserLoginModal.show({
-				origin: origin,
-				callback: function () {
-					w.UserLogin.forceLoggedIn = true;
-					cb();
-				}
+			require(['AuthModal'], function (authModal) {
+				authModal.load({
+					origin: origin,
+					url: '/signin',
+					successAuthCallback: function () {
+						w.UserLogin.forceLoggedIn = true;
+						cb();
```

Comment:
Wow, this ```cb``` name is really confusing. 
@RafLeszczynski I thought we avoid using acronyms. Are there any plans to refactor this to follow our guidelines?